### ISSUE TITLE
German Scheme Feedback:  Identify the SFR Impact on Cloud Deployment

### ISSUE THREAD

**Issue by jgb1128** [CC-in-the-Cloud/Admin#154]
Under 3rd bullet, 2nd sub bullet: "It may be the case that a requirement simply does not apply to a cloud environment, in
which case the PP author could consider defining that requirement as an **implementation based
SFR** that applies only to the on-premise use case."  _Is this the same as selection-based SFR as in ND_cPP? Using PP-Modules could also be an option in this case._

**Comment by bharveyTX** [comment]
Yes, either the higher level mandatory SFR could include a selection for the on-prem specific SFR or you would have to modify the SFR to have selections for on-prem or cloud itself.

**Comment by jgs1618** [comment]
I think one ought to be careful of using the phrase "implementation based SFR".  Section 7.3.2.1 of CC:2022 Part 1 states "In the context of PPs and PP-Modules, the SFRs shall be independent of any specific technical solution (implementation)."

The original text probably needs to be rewritten as selections, where one selection is if it a cloud.

**Comment by bharveyTX** [comment]
This issue is mostly a question from BSI which has been addressed. No changes needed to implement in Guidance. Closing.
